network-store-model/src/main/java/com/powsybl/network/store/model/ExtensionAttributes.java

```diff
@@ -18,4 +19,10 @@
 )
 @JsonTypeIdResolver(ExtensionAttributesIdResolver.class)
 public interface ExtensionAttributes {
+    // This property is used to not persist some extensions that are only used at import/export.
+    // This workaround can be removed when private importers do not add private extensions to build open source classes.
```

Comment:
```suggestion
    // This property is used to not persist some extensions that are only used at import/export.
    
```
I suggest to remove the second comment as it's not so clear ?

Reply:
Yes ok. It's about private code